### PR TITLE
feat(logs-placeholder): update placeholder for Logs

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-797

- Add placeholder for DB Managed on the Live logs
- Refacto placeholders and using only one component for all Logs 
- Don't hesitate to re-test different case because the placeholders logics are modified 

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 937d87a0-2391-4f54-8dbb-e263f2b05c0e | [test-old-status](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/937d87a0-2391-4f54-8dbb-e263f2b05c0e/services/general) |
 /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Take rest sometimes

File: libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx [hunk 1]
```diff
@@ -122,24 +132,27 @@ export function LayoutLogs(props: PropsWithChildren<LayoutLogsProps>) {
             'Live logs',
             ENVIRONMENT_LOGS_URL(organizationId, projectId, environmentId) + SERVICE_LOGS_URL(serviceId),
             environmentId,
-            serviceId
+            serviceId,
+            (service as DatabaseEntity)?.mode === DatabaseModeEnum.MANAGED ? false : true
```
Comment (ctjhoa) [inline libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx:136]:
```suggestion
            (service as DatabaseEntity)?.mode !== DatabaseModeEnum.MANAGED
```

**Review by ctjhoa — CHANGES_REQUESTED** [review]

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://790-console-test-old-status.zc531a994.rustrocks.cloud)

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**